services/121-service/src/sms/sms/sms.service.ts

```diff
@@ -0,0 +1,68 @@
+import { STAGING_URL } from './../../config';
+import { Injectable } from '@nestjs/common';
+import { TWILIO } from '../../secrets';
+import { DEBUG, PRODUCTION_URL } from '../../config';
+import { twilioClient } from './twilio.client';
+import { TwilioMessageEntity } from './twilio.entity';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+
+let appUrl: string;
+if (process.env.NODE_ENV == 'production') {
```

Comment:
This 'juggling' of the `appUrl` should all happen in the config-file, based on the ENV-variables used there.